docs/operations.md

```diff
@@ -123,6 +165,110 @@ stays in the database:
+| YAML                 | `ScavengeHashUsersCacheCapacity`       |
+| Environment variable | N/A                                    |
+
+## HTTP option
+
+### Start From Chunk (_Deprecation Notice_)
+
+### Thread
+
+Specify the number of threads to use for running the scavenging process. The default value is 1.
+
+_Thread must be 1 or above_
+
+```
+Ex: POST to http://127.0.0.1:2113/admin/scavenge?thread=2
+```
+
+### Threshold
+
+By default, when running a scavenge, all chunks related to the events that need to be scavenged are removed, and new chunks are created. Setting this option allows you to optimize the chunk weight threshold for scavenging. This optimization helps in resource and time efficiency.
+
+Possible values for the threshold:
+
+-1: Scavenge all chunks, even if there is nothing to scavenge in them.
+
+0: Default value, scavenges every chunk that has something to scavenge in it.
+
+&gt; 0: The minimum weight a chunk must have to be scavenged.
+
+```
+Ex: POST to http://127.0.0.1:2113/admin/scavenge?threshold=2000
+```
+
+### Throttle percent
+
+The scavenging process can be time-consuming and resource-intensive. You can control the speed and resource usage of the scavenge process using the throttlePercent option. When set to 100 (default value), the scavenge process runs at full speed. Setting it to 50 makes the process run twice as long by pausing regularly in between, consuming half the resources of the server.
+
+_Throttle percent must be between 1 and 100._
+
+_Throttle percent must be 100 for a multi-threaded scavenge._
+
+```
+Ex: POST to http://127.0.0.1:2113/admin/scavenge?throttlePercent=50
+```
+
+### Sync Only
+
+This option is a boolean value. When set to true, it prevents the creation of a new scavenge point. The scavenge process only run if there is an exist scavenge point that has not been used yet.
+
+The following actions are performed:
+* If there is no latest scavenge point, the process exits.
+* If the latest scavenge point exists and already in use, the process exits.
+* If the latest scavenge point exists and has not been used yet,the scavenge process runs on that scavenge point.
+
+```
+Ex: POST to http://127.0.0.1:2113/admin/scavenge?syncOnly=true
+```
+
+### Best Practices
+
+1. _Plan and schedule_: Create a plan for when and how frequently you will run the scavenging process. Consider factors such as system load and usage pattern to determine the optimal schedule for scavenging. It is generally advisable to perform scavenging during periods of low activity to minimize potential disruptions, especially when running it for the first time.
+2. _Run scavenge on follower node_: Running scavenging on a follower node helps reduce the workload on the leader. After the process, information will be sent back to the leader.
+3. _Back up data_: It is important to have a backup before running scavenging in case any unforeseen issues or data loss occurs during the process.
```

Comment:
It is important to have a backup before running scavenging in case any unforeseen issues or data loss occurs during the process.

->

Do not take backups while scavenging is running. You can use the /admin/scavenge/current endpoint to find out if scavenging is running.